app/src/main.py

```diff
@@ -122,7 +121,4 @@ async def main():
     await vehicle_app.run()
 
 
-LOOP = asyncio.get_event_loop()
-LOOP.add_signal_handler(signal.SIGTERM, LOOP.stop)
-LOOP.run_until_complete(main())
-LOOP.close()
+asyncio.run(main())
```

Comment:
And another one i just found. please add a new line to the end :)

Reply:
There is new line on the end, but somehow is not visible here.

Reply:
ok than i didnt say anything :)